intro.ipynb

```diff
@@ -1,99 +1,179 @@
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "import matplotlib.pyplot as plt\n",
-    "import numpy as np\n",
-    "%matplotlib inline"
+    "## Welcome! 🐍\n",
+    "\n",
+    "Hello and welcome, everyone! We’re excited that you chose to embark in this Python journey\n",
+    "\n",
+    "Over **four** days, we'll see a lot of Python. **Two** fundamental topics each day\n",
+    "\n",
+    "::: {.panel-tabset}\n",
+    "\n",
+    "### Day 1\n",
+    "1. Basic syntax and data types\n",
+    "2. Control flow, loops, and exceptions\n",
+    "\n",
+    "### Day 2\n",
+    "1. Functions\n",
+    "2. Working with files\n",
+    "\n",
+    "### Day 3\n",
+    "1. Functional programming\n",
+    "2. Object-oriented programming\n",
+    "\n",
+    "### Day 4\n",
+    "1. Modules and packages\n",
+    "2. Questions, exercises, more questions...\n",
+    "\n",
+    ":::\n",
+    "\n",
+    "\n",
+    "::: {.notes}\n",
+    "\n",
+    "Although this is an introductory course, we will be going quite in-depth to cover as much ground as possible. Our aim is to equip you with a strong foundation in Python programming that you can build on in the future. So, let's get started!\n",
+    "\n",
+    ":::"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "plt.scatter(np.random.randn(1000),\n",
-    "            np.random.randn(1000),\n",
-    "            c=np.random.randn(1000), s=100)"
+    "## Timetable\n",
+    "\n",
+    "**TL;DR:** Every morning, 9:00-13:00, on Zoom\n",
+    "\n",
+    ". . .\n",
+    "\n",
+    "More in detail:\n",
+    "\n",
+    "::: {.incremental}\n",
+    "\n",
+    "- 8:45-9:00 — Welcome, warm-up questions\n",
+    "- 9:00-10:50 — Topic 1 (theory, hands-on, Q&A)\n",
+    "- *10:50-11:10 — Break ☕*\n",
+    "- 11:10-13:00 — Topic 2\n",
+    "\n",
+    ":::"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "from tutorial.common import Question, Quiz"
+    "## How to interact\n",
+    "\n",
+    "::: {.notes}\n",
+    "\n",
+    "Each block is 20% theory and 80% hands-on. It's an online workshop, so interaction between us is **essential**\n",
+    "\n",
+    ":::\n",
+    "\n",
+    "A few tips to interact:\n",
+    "\n",
+    "::: {.panel-tabset}\n",
+    "\n",
+    "### On Zoom\n",
+    "- Raise your hand 🤚 then ask\n",
+    "- Write in the chat\n",
+    "- We will open a break-out room to address longer questions. **Anyone** interested can jump in\n",
+    "\n",
+    "\n",
+    "### On Slack\n",
+    "- Ask your question in the **corresponding channel**. Use `#help` for generic questions\n",
+    "- Reply **in a thread** if you want to add a comment on another question\n",
+    "- Use `@Sasha`, `@Simone`, `@Edoardo` if you want to mention the tutorial instructors directly\n",
```

Comment:
I think Simone specified their last name too. Another option is to change the name to `Simone` on slack.

```suggestion
    "- Use `@Sasha`, `@Simone Baffelli`, `@Edoardo` if you want to mention the tutorial instructors directly\n",
```

Reply:
I changed my name, if you want you can just use `@Simone` now